src/overlays/actors/ovl_En_Bx/z_en_bx.c

```diff
@@ -182,18 +182,15 @@ void EnBx_Update(Actor* thisx, GlobalContext* globalCtx) {
     }
 }
 
-#ifdef NON_MATCHING
-// Issue with Gfx_TwoTexScroll I think
 void EnBx_Draw(Actor* thisx, GlobalContext* globalCtx) {
     EnBx* this = THIS;
-    void* matrices = Graph_Alloc(globalCtx->state.gfxCtx, 4 * sizeof(Mtx));
+    Mtx* matrices = Graph_Alloc(globalCtx->state.gfxCtx, 4 * sizeof(Mtx));
```

Comment:
Considering this temp is being incremented afterwards now, I don't think it should be called `matrices`.
```suggestion
    Mtx* mtx = Graph_Alloc(globalCtx->state.gfxCtx, 4 * sizeof(Mtx));
```